services/order.py

```diff
@@ -0,0 +1,31 @@
+from db.models import Ticket, Order, User, MovieSession
+from django.db import transaction
+from django.db.models.query import QuerySet
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> None:
+    with transaction.atomic():
+        customer_user = User.objects.get(username=username)

```

Comment:
Use get_user_model() instead of User. Check out the [checklist](https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency). Fix it everywhere)

Reply:
i will aprove, but fix it here also)